src/build_backend.py

```diff
@@ -11,7 +11,8 @@
 
 from cockpit import __version__
 
-PACKAGE = f'cockpit-{__version__}'
+VERSION = __version__ or '0'
```

Comment:
Not sure how important that is, but it could do what autogen.sh/Makefile.am do, and fall back to `git describe`?

For sure this is follow-up material, let's not stall this PR on that.